ledger/ledger-api-domain/src/main/scala/com/digitalasset/ledger/api/DeduplicationPeriod.scala

```diff
@@ -73,7 +73,9 @@ object DeduplicationPeriod {
     )
   }
 
-  /** The `offset` defines the start of the deduplication period. */
+  /** The `offset` defines the start of the deduplication period.
```

Comment:
I think adding the same `exclusive` suffix should suffice

Reply:
Good idea.